spirit_simulator/a1_description/package.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<package format="2">
+    <name>a1_description</name>
+    <version>0.0.0</version>
+    <description>The a1_description package</description>
+
+    <maintainer email="jimingre@andrew.cmu.edu">unitree</maintainer>
+    <license>TODO</license>
```

Comment:
Is this meant to be left as a TODO?

Reply:
Likewise, we should check the license on this package to see if there is any alterations we need to make, although that could likely be a separate PR.